stats.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Automattic\VIP\Stats;
+
+// Limit tracking to production
+if ( false !== VIP_GO_ENV ) {
+	add_action( 'transition_post_status', __NAMESPACE__ . '\track_publish_post', 9999, 10, 2 );
+}
+
+/**
+ * Count publish events regardless of post type
+ */
+function track_publish_post( $new_status, $old_status ) {
+	if ( 'publish' !== $new_status || 'publish' === $old_status ) {
+		return;
+	}
+
+	$pixel = add_query_arg( array(
+		'v'                     => 'wpcom-no-pv',
+		'x_vip-go-publish-post' => FILES_CLIENT_SITE_ID,
+	), 'http://pixel.wp.com/b.gif' );
+
+	wp_remote_get( $pixel, array( 'blocking' => false, ) );
```

Comment:
Can't hurt. Tracks does: https://github.com/Automattic/jetpack/blob/d40db24971bec4fe0355d909fbe0a6ac6ce972d9/_inc/lib/tracks/class.tracks-client.php#L81. It also makes a blocking request, but I'm not sure we need to, unless we move this to the async hook coming with #471.